spec/omniauth/strategies/untappd_spec.rb

```diff
@@ -81,7 +81,7 @@
       it { expect(info[:name]).to eql 'John Doe' }
       it { expect(info[:nickname]).to eql 'john_doe' }
       it { expect(info[:email]).to eql 'john@doe' }
-      it { expect(info[:urls]['Untappd']).to eql 'http://untappd.com/user/john_doe' }
+      it { expect(info[:urls]['Untappd']).to eql 'https://untappd.com/user/john_doe' }
```

Comment:
Line is too long. [86/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.